tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_schema.go

```diff
@@ -0,0 +1,104 @@
+package dataapigeneratorjson
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+	"github.com/hashicorp/pandora/tools/sdk/resourcemanager"
+)
+
+type SchemaModel struct {
+	Fields []SchemaField `json:"Fields"`
+}
+
+type SchemaField struct {
+	HclName       string                           `json:"HclName"`
+	Type          string                           `json:"Type"`
+	Documentation *string                          `json:"Documentation,omitempty"`
+	Required      *bool                            `json:"Required,omitempty"`
```

Comment:
ObjectDefinition was removed but I think this is still needed. Perhaps we leave it as a TODO and take a look at it together?

Reply:
Got it figured out and added!